unstructured/partition/pdf.py

```diff
@@ -761,7 +761,7 @@ def check_coords_within_boundary(
         a float ranges from [0,1] to scale the horizontal (x-axis) boundary
     """
     if not coord_has_valid_points(coordinates) and not coord_has_valid_points(boundary):
-        raise ValueError("Invalid coordinates.")
+        return False
```

Comment:
worth adding a trace detail message i think, like: https://github.com/Unstructured-IO/unstructured/blob/9a3e24fcbb4efef30d5f046952e794666345b7ea/unstructured/partition/utils/sorting.py#L70
.
is there a single page this fails on that could be added to a test? one could extract the problematic page with something like:

    qpdf --empty --pages orignal.pdf 89 -- ~/tmp/docs/single-page-p89.pdf

finally, does not need to be addressed in this PR, but what is the why for the invalid coordinate? is it negative values? (which i think are actually legit in PDF's in certain cases)

Reply:
Sounds good - I'll add trace detail and a test!

And yes - the values that it was erroring on were negative.

Reply:
here's where it was failing: https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/partition/utils/sorting.py#L30